README.md

```diff
@@ -66,15 +67,15 @@ Refer [here](https://github.com/actions/checkout/blob/v1/README.md) for previous
 ## Checkout a different branch
 
 ```yaml
-- uses: actions/checkout@preview
+- uses: actions/checkout@v2-beta
```

Comment:
i also have a pr to fix this: https://github.com/actions/checkout/pull/90

many users seem to be interested in checking out the HEAD commit of a PR rather than the merge commit. Seemed like an example was warranted (since it's otherwise not intuitive how to achieve it)